src/packages/excalidraw/webpack.prod.config.js

```diff
@@ -68,6 +69,7 @@ module.exports = {
             loader: "file-loader",
             options: {
               name: "[name].[ext]",
+              outputPath: "excalidraw-assets",
```

Comment:
doesn't this mean that non-font chunks won't be put into `excalidraw-assets` folder? E.g. `vendor.js`.

Reply:
All chunks will go inside `excalidraw-assets` now including fonts so users need not manually copy anything and just copy `excalidraw-assets` as it is.
<img width="839" alt="Screenshot 2020-12-08 at 2 25 27 AM" src="https://user-images.githubusercontent.com/11256141/101404460-acfb5a80-38fc-11eb-970c-1fca88da0cd8.png">

Reply:
Alright